pkgs/top-level/all-packages.nix

```diff
@@ -9311,6 +9311,8 @@ with pkgs;
 
   mjolnir = callPackage ../servers/mjolnir { };
 
+  draupnir = callPackage ../servers/draupnir {};
+
```

Comment:
ideally, this would be sorted (in its category), but this is a lost cause by now.

Reply:
Any suggestions where I'd put it then?